apps/www/package.json

```diff
@@ -3,14 +3,14 @@
   "version": "0.0.1",
   "private": true,
   "scripts": {
-    "dev": "next dev -p 3001",
+    "dev": "concurrently \"contentlayer dev\" \"next dev\"",
```

Comment:
I don't think we need this anymore. `next dev` should kick off the contentlayer plugin.

Reply:
okay, sounds good